src/pages/Engineering.pug

```diff
@@ -70,6 +68,23 @@ block article
 
         # Preventing secondary structures
 
+        Due to the formation of duplex secondary structures, especially for guanine, elongation by TDT is enormously complicated. 
+        This problem often prevents the extension of all strands to more than 1000 nt. 
+        By adding a single-stranded binding protein (SSB)  available from Thermo Fisher Scientific, we were able to show that SSBs can be used to partially prevent the formation of secondary structures. 
+        As a result, significantly longer strands could be amplified.
+    
+    - var imgpath = "" + require("../assets/img/Experiments/GC_oligo_ohne_SSBs.png")["default"]
+    +image(4, imgpath, "Evaluation of the sequencing of inserts from the approach without SSBs. It can be seen that very few nucleotides were appended due to the secondary structures present. ")
+    
+     - var imgpath = "" + require("../assets/img/Experiments/GC_Oligo_mit_SSB.png")["default"]
+    +image(5, imgpath, "Analysis of the sequencing of inserts that were elongated with SSBs. It can be seen that here clear extensions have taken place with one primer even by 894 nucleotides.")
+
+    :markdown-it(html)
+    
+        In summary, for secondary structures, SSBs can help to avoid them, which allows GC-rich regions to be extended. 
```

Comment:
```suggestion
        As a conclusion from our experiments it can be seen that we were able to reduce the formation of secondary structures in ssDNA by using SSBs. Adding them to our TdT tailing apporaches significantly enhanced the elongation capacity. 
```